api/gpt.py

```diff
@@ -10,9 +10,13 @@ def set_openai_key(key):
 class Example():
     """Stores an input, output pair and formats it to prime the model."""
 
-    def __init__(self, inp, out):
+    def __init__(self, inp, out, input_prefix = "input: ", input_suffix = "\n", output_prefix = "output: ", output_suffix = "\n"):
```

Comment:
To minimize redundancy, I think it might be better to declare the prefixes and suffixes in the `GPT` class, and remove the `format` method from this class entirely. See my comment in the `GPT` class for an idea of how to format the example in there.

Reply:
I've removed the formate function entirely, and all references to the prefix from Example.  I've added a new format_example function to GTP as suggested which is now called by the add_example function.

Reply:
Awesome!